recipes/glad/all/conanfile.py

```diff
@@ -1,25 +1,25 @@
 import os
 
-from conans import ConanFile, CMake, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, collect_libs, copy, export_conandata_patches, get, rmdir
+from conan.errors import ConanInvalidConfiguration
 
 class GladConan(ConanFile):
     name = "glad"
     description = "Multi-Language GL/GLES/EGL/GLX/WGL Loader-Generator based on the official specs."
-    topics = ("conan", "glad", "opengl")
+    topics = ("glad", "opengl")
```

Comment:
The name is not needed since it does not help with searching :)

```suggestion
    topics = ("opengl")
```

Reply:
there must be a trailing comma, otherwise it's not a tuple for python

Reply:
fixed :)